packages/lit-apollo/apollo-query.d.ts

```diff
@@ -1,4 +1,4 @@
-import { ApolloQuery as ApolloQueryBase } from "@apollo-elements/mixins/apollo-mutation-mixin";
+import { ApolloMutation as ApolloQueryBase } from "@apollo-elements/mixins/apollo-mutation-mixin";
```

Comment:
Why should this be ApolloMutation? it's the query mixin.. probably should be
```suggestion
import { ApolloQuery as ApolloQueryBase } from "@apollo-elements/mixins/apollo-query-mixin";
```